Add working group descriptions and links (#57)
hyphacoop/handbook: working-groups.md

# Working Groups 

### [Business Planning][biz-wg]

WG for stewarding our business strategy, this includes generating and pursuing business opportunities, relationships, branding, and communications.

📅 Calls: [Weekly][calendar]  
👪 Members: [**@business-planning-wg**][t-biz]  
✅ Tasks: GitHub board > [`wg:business-planning` label][l-biz]  

### [Finance][fin-wg]

WG for stewarding the financial sustainability of the organization, this includes establishing accounting practices, maintaining financial accounts, processing invoices, expenses, and payroll, and creating reports on our financial statements and tax obligations.

📅 Calls: [Weekly][calendar]   
👪 Members: [**@finance-wg**][t-fin]  
✅ Tasks: GitHub board > [`wg:finance` label][l-fin]  

### [Governance][gov-wg]

> WG for drafting by-laws, articles of incorporation, and planning for the first General Meeting (and subsequent structure).

Author: Slightly more current take: 
```suggestion
WG for ensuring healthy structure of our cooperative, this includes drafting by-laws, maintaining articles of incorporation, and planning annual members meetings.
```

Author: Okay, per @benhylau, I'll merge in governance, merge this PR, and expect infra to update shortly

> 📅 Calls: [Biweekly][calendar]  

Reviewer: Looks pretty weekly to me :D

Author: I think we're weekly for this three weeks, but when I'm looking at the calendar ahead it moves back to biweekly? (We can confirm on Wednesday -- but that seems like the right rhythm for governance)

👪 Members: [**@governance-wg**][t-gov]  
✅ Tasks: GitHub board > [`wg:governance` label][l-gov]  

### [Infrastructure][inf-wg]

WG for setting up our "virtual office", in particular the infrastructure and processes to collaborate.

📅 Calls: [Biweekly-ish][calendar]   

> 👪 Members: [**@infra-wg**][t-inf]  

Reviewer: ```suggestion
👪 Members: [**@infrastructure-wg**][t-inf]  
```

> ✅ Tasks: GitHub board > [`wg:infra` label][l-inf]  

Reviewer: ```suggestion
✅ Tasks: GitHub board > [`wg:infrastructure` label][l-inf]  
```

### [Operations][ops-wg]

WG for stewarding our shared practices and policies for effective remote collaboration.

📅 Calls: [Biweekly][calendar]  

> 👪 Members: [**@ops-wg**][t-ops]  

Reviewer: ```suggestion
👪 Members: [**@operations-wg**][t-ops]  
```

> ✅ Tasks: GitHub board > [`wg:ops` label][l-ops]  

Reviewer: ```suggestion
✅ Tasks: GitHub board > [`wg:operations` label][l-ops]  
```

<!-- Links: Important -->
[calendar]: https://link.hypha.coop/calendar

<!-- Links: Working Groups -->
[biz-wg]: https://link.hypha.coop/biz-wg
[fin-wg]: https://link.hypha.coop/fin-wg
[gov-wg]: https://link.hypha.coop/gov-wg
[ops-wg]: https://link.hypha.coop/ops-wg
[inf-wg]: https://link.hypha.coop/inf-wg

[l-biz]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:business-planning"
[l-fin]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:finance"
[l-gov]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:governance"

> [l-ops]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:ops"

Reviewer: ```suggestion
[l-ops]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:operations"
```

> [l-inf]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:infra"

Reviewer: ```suggestion
[l-inf]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:infrastructure"
```

[t-biz]: https://github.com/orgs/hyphacoop/teams/business-planning-wg/members
[t-fin]: https://github.com/orgs/hyphacoop/teams/finance-wg/members
[t-gov]: https://github.com/orgs/hyphacoop/teams/governance-wg/members 
[t-ops]: https://github.com/orgs/hyphacoop/teams/ops-wg/members

> [t-inf]: https://github.com/orgs/hyphacoop/teams/infra-wg/members

Reviewer: ```suggestion
[t-ops]: https://github.com/orgs/hyphacoop/teams/operations-wg/members
[t-inf]: https://github.com/orgs/hyphacoop/teams/infrastructure-wg/members
```

# Working Groups 

### [Business Planning][biz-wg]

> WG for performing feasibility study, and business planning to identify start up costs, fundraising mechanisms, and risks.

Reviewer: Also sales and grants now!

Author: Yes! we should revise

Reviewer 2: Could we more explicitly mention activities that are more generally recognizable as what we've aspired to? Like does any of this sound appropriate:

- writing/updating business plan
- sales and grant application pipelines and support
- communications and outreach (incl social media)
- marketing

Reviewer: Ok I take a stab:

```suggestion
WG for strategizing the business goals of the organization, this includes branding and marketing communications, building relationships with clients and neighbours, supporting project teams in sales and grant initiatives, and situating ourselves in the market and solidarity economy toward our organizational mission.
```

Reviewer 2: Sure! Thanks. Thoughts on using bullets for these WG descriptions instead of paragraphs, so verbs and nouns kinda line up, and sub-items are more clear?

(I was planning to lean on bullets when we get to infra's)

Author: I prefer sentence over bullet form for something like a high-level description (which is what I took these to be). More detail can be included below. But I think we need a 1 sentence description for each.

Author: @benhylau can we discuss this as bizdev? I have many suggestions to streamline the writing and I think it'll be faster synchronously

Author: Discussed as a group, this is our proposed revision:
```suggestion
WG for stewarding our business strategy, this includes generating and pursuing business opportunities, relationships, branding, and communications.
```

> 📅 Weekly calls  

Reviewer: This seems duplicate info as calendar and will fall out of sync

Author: I think adding the date would create that issue, but I'm not seeing "weekly" as a major concern, as we've kept to a pretty regular meeting schedule since each group has started? 

Could have these link to the calendar all anyway so that remains the primary source?

Author: Will add link to calendar as a way to resolve this.

> 🌱 Cultivators:   

Reviewer: This seems unnecessary as it changes all the time

Author: I'm agnostic on this, was on loomio but yes, we could remove

Reviewer 2: agree with ben. would prefer to have all these roles in a single org-wide spreadsheet, maybe with monthly granularity (publiclab.org does something like this for chairing/running projects)

Author: removing

👪 Members: [**@business-planning-wg**][t-biz]  
✅ Tasks: GitHub board > [`wg:business-planning` label][l-biz]  

### [Finance][fin-wg]

> WG for establishing accounting practices, including setting up a bank account, administering payroll and expenses, invoicing, and handling remittances.

Reviewer: ```suggestion
WG for stewarding the financial sustainability of the organization, including establishment of accounting practices, maintenance of our financial accounts, facilitation of invoicing, expenses, and payroll, and reporting of our financial statements and tax obligations.
```

Author: Makes sense! However, there isn't agreement in this list, can I propose a friendly amendment (Allll about grammar):

```suggestion
WG for stewarding the financial sustainability of the organization, this includes establishing accounting practices, maintaining financial accounts, processing invoices, expenses, and payroll, and creating reports on our financial statements and tax obligations.
```

Reviewer 2: Maybe after we commit these, we could toss it back to WG's to revisit and amend before some deadline? We should prob get in the habit of updating these through more of a shared process (instead of PR), and now seems like as good a time as ever :)

Author: Yes, the intent is not that @hyphacoop/ops-wg does this, just making space for each group to contribute

Reviewer: Thanks for fixing this @dcwalk much better.

📅 Weekly calls  
🌱 Cultivators:  
👪 Members: [**@finance-wg**][t-fin]  
✅ Tasks: GitHub board > [`wg:finance` label][l-fin]  

### [Governance][gov-wg]

WG for drafting by-laws, articles of incorporation, and planning for the first General Meeting (and subsequent structure).

📅 Biweekly calls  
🌱 Cultivators: dc  
👪 Members: [**@governance-wg**][t-gov]  
✅ Tasks: GitHub board > [`wg:governance` label][l-gov]  

### [Infrastructure][inf-wg]

WG for setting up our "virtual office", in particular the infrastructure and processes to collaborate.

📅 Biweekly(ish) calls  
🌱 Cultivators: patcon & elon. (next rotation after..?)  
👪 Members: [**@infra-wg**][t-inf]  
✅ Tasks: GitHub board > [`wg:infra` label][l-inf]  

### [Operations][ops-wg]

WG for stewarding our shared practices and policies for effective remote collaboration.

📅 We host biweekly calls  
🌱 Cultivators:   
👪 Members: [**@ops-wg**][t-ops]  
✅ Tasks: GitHub board > [`wg:ops` label][l-ops]  

> <!-- Links: Working Groups -->

Reviewer: Am I supposed to change all these shortlinks to fully written out? `https://link.hypha.coop/business-planning`

Author: no, I hope not! I see these more as internal, vs. ones used by people (@patcon tell me I'm wrong?)

Reviewer 2: Either sounds fine! Don't feel need to change, but def no objection to it (i wasn't imagining we were deleting/changing these shortlinks tho, but more like append only)

Reviewer: I don't think they are append only in general as I can imagine times when we want to retire things, but in this case I think we can add redundant, like:

- https://link.hypha.coop/biz-wg
- https://link.hypha.coop/business-planning -> biz-wg

Because it's hard to remember all the short forms, and whether it's biz-wg or wg-biz or biz...
Anyhow I don't use this, so leaving as is for now.

Reviewer 2: Maybe we could worry about that when we near the 500 shortlink limit? (rebrandly keeps stats, so we can see what's unused when that time comes) There's just no need for that list to be clean or sorted imho

Author: resolving as it seems like not pertinent to this PR

[biz-wg]: https://link.hypha.coop/biz-wg
[fin-wg]: https://link.hypha.coop/fin-wg
[gov-wg]: https://link.hypha.coop/gov-wg
[ops-wg]: https://link.hypha.coop/ops-wg
[inf-wg]: https://link.hypha.coop/inf-wg

[l-biz]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:business-planning"
[l-fin]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:finance"
[l-gov]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:governance"

> [l-ops]: https://github.com/orgs/hyphacoop/projects/2?card_filter_query=label:"wg:ops"

Reviewer: We discussed to not change GitHub labels and team names, but looking at this it seems `wg:ops` and `wg:infra` should be written out as:

- `wg:ops` -> `wg:operations`
- `wg:infra` -> `wg:infrastructure`

Can I make the change across the org?

Author: I brought this up in the Ops call yesterday `2020-03-11`, I think these could be expanded to be consistent...

That said, I think "infra" and "ops" are used so heavily in many domains consistently to mean that thing that I think we could get by without.

My take on an approach: least possible change for an elegant solution.

Reviewer 2: > our guideline is to have written out names

just noting that i'm unaware of this guideline :)

Author: I brought up in ops call we have human-readable written out names for labels (and in other places where humans read things), I didn't want to lose that for three letter short codes like those that are present in the links below.

@benhylau noticed we had like four ways of referring to each group, and wants to make them more consistent, which I fully support:
- full names (e.g., finance, governance)
- hybrid abbrev (e.g., bizdev)
- standard abbrev (e.g., infra, ops)
- 3-letter (e.g., inf, gov, fin)

We haven't been using those the same way in the same places. With @benhylau 's edits to template we are much more consistent. I think using a mix of:
- full names (e.g., finance, governance) for things people read (tho "infra" and "ops)
- 3-letter (e.g., inf, gov, fin) for things like links

This works for me, but if Ben would like to see through **TOTAL CONSISTENCY** I'm fine with that.

Reviewer: Okay I am going to change everything to suit this policy:

- Internal references -> "3-letter (e.g., inf, gov, fin) for things like links"
- Public and visible -> "full names (e.g., finance, governance)"
